### ISSUE TITLE
Push interface for issuing queries

### ISSUE THREAD

**Issue by ghost** [mami-project/pto-postgres#35]
When submitting a query, don’t link directly to the result page. Instead, show an animation that runs for 10 seconds and if the query isn’t done in that time, show a form field where users can enter an email address. Once the query is done, users get an email to that email address. (May not be done by 1 March.)

**Comment by ghost** [comment]
WONTFIX
